client/src/components/Reviews/components/ReviewList.jsx

```diff
@@ -30,8 +31,19 @@ const MoreReviewsButton = styled.button`
   bottom:  2%;
   cursor: pointer;
 `;
+const StyledSelect = styled.select`
+  border: none;
+`;
 
 function ReviewList() {
+  const [currentSort, setCurrentSort] = useState('relevant');
+  useEffect(() => {
+    // TODO axios.get() with currentSort param
+  }, [currentSort]);
+  function HandleSort(e) {
```

Comment:
I think this and your other function should be camel cased. React assumes that everything that starts with a capital letter is a react component, so this may cause some issues.